components/GroupSearchFilters/GroupSearchFilters.styles.js

```diff
@@ -0,0 +1,21 @@
+import styled, { css } from 'styled-components';
+import { themeGet } from '@styled-system/theme-get';
+
+import { system } from 'ui-kit';
+
+const GroupSearchFilters = {};
+
+const visibility = ({ visible }) => css`
+  visibility: ${visible ? 'visible' : 'hidden'};
```

Comment:
@rkdavidson Should we add `display: none` here too. Otherwise the `p` tag still takes up space on the page.
![image](https://user-images.githubusercontent.com/2528817/108239791-df50e180-710f-11eb-9f08-0d51d4f5bf55.png)